app/views/docs/functions.phtml

```diff
@@ -211,7 +211,7 @@ void main() { // Init SDK
         <?php foreach($environments as $key => $environment): ?>
         <tr>
             <td><img src="" data-ls-attrs="src=/images/environments/<?php echo $this->escape($environment['logo'] ?? ''); ?>" alt="Function Env." class="avatar xxs" /></td>
-            <td><?php echo $this->escape($environment['name'] ?? ''); ?> <?php echo $this->escape($environment['version'] ?? ''); ?></td>
+            <td><span data-general-copy><?php echo $key ?> </span></td>
```

Comment:
Although we're not consistent enough, it is a better practice to use `$this->escape()` to echo vars inside the phtml templates.

Reply:
Addressed in 3ab7ae8